public/download.php

```diff
@@ -9,64 +14,71 @@
 <div id="mainpage">
     <div id='fullcontainer'>
 
-        <h2>Downloads</h2>
-
-        <p class="embedded mb-5">
-            <b>Upgrade:</b> If you were brought to this page because the emulator told you
-            that a new version is available, download the new version and
-            extract it over the existing folder. This way you won't lose
-            any save files that you may have.
-        </p>
+        <h2>Emulators supporting RetroAchievements</h2>
```

Comment:
Not sure this is a good title for the page. The menu item still says "Download.

Reply:
"Download an Emulator" is the only alternative I can come up with, but it loses the context that these are special emulators that support RetroAchievements. That may be implied given this is on the RA website.

This page probably doesn't matter as much for mobile, but "Emulators supporting RetroAchievements" is going to wrap to two lines on most phones, whereas "Download an Emulator" doesn't, at least for an iPhone X. "Download an Emulator" also aligns with the menu item copy.